tests/lca/imagebasedupgrade/mgmt/upgrade/tests/e2e-upgrade-test.go

```diff
@@ -798,6 +811,39 @@ func updateIBUWithCustomCatalogSources(imagebasedupgrade *lca.ImageBasedUpgradeB
 	}
 }
 
+func validateDualStackDeployment() (bool, bool) {
```

Comment:
I think this function should just return true if the cluster is dualstack and false if it isn't